test_test.go

```diff
@@ -215,4 +215,4 @@ func testInnerTests(t *testing.T) {
 	if test.Tests[2].Error == nil || !strings.Contains(test.Tests[2].Error.Error(), "status") {
 		t.Errorf("expected must status to fail")
 	}
-}
+}
```

Comment:
Sure thing. I may have inadvertently removed that trailing whitespace. I'll add it in as soon as I get back home. Thanks for a prompt review!